src/web/app/pom.xml

```diff
@@ -1594,7 +1594,7 @@
       <id>status-monitoring</id>
       <dependencies>
         <dependency>
-          <groupId>org.geoserver.community</groupId>
+          <groupId>org.geoserver.extension</groupId>
```

Comment:
If the module comes to core this shoudl go away.

Reply:
Removed, thanks.